pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -110,8 +110,8 @@ def endpoint_pieces(cls):
                 except AttributeError:
                     # no endpoint_name defined, need to get more specific in the MRO
                     continue
-
-            pieces = (master_endpoint_name, cls.endpoint_name)
+            pieces = tuple(cls.endpoint_name.split('/'))
```

Comment:
allows plugins to namespace tasks.

v3/tasks/core-update/ becomes tasks-core-update-list and tasks-core-update-detail